content/en/tracing/trace_collection/library_config/nodejs.md

```diff
@@ -48,6 +48,15 @@ The version number of the application.
 **Default**: `{}`<br>
 Set global tags that are applied to all spans and runtime metrics. When passed as an environment variable, the format is `key:value,key:value`. When setting this programmatically, the format is `tracer.init({ tags: { foo: 'bar' } })`.
 
+`DD_TRACE_HEADER_TAGS`
+: **Configuration**: `headerTags` <br>
+Accepts a map of case-insensitive header keys to tag names and automatically applies matching header values as tags on traces. Also accepts entries without a specified tag name that are automatically mapped to tags of the form `http.request.headers.<header-name>` and `http.response.headers.<header-name>` respectively.<br><br>
```

Comment:
```suggestion
Accepts a map of case-insensitive header keys to tag names and automatically applies matching header values as tags on traces. Also accepts entries without a specified tag name that are automatically mapped to tags of the form `http.request.headers.<header-name>` and `http.response.headers.<header-name>`.<br><br>
```

Reply:
Feel free to ignore this suggestion if I'm wrong here, but it seems like there's only one thing (`entries without a specified tag name`) that's being referenced by the response and request headers. 

Let me know if I'm missing something here!

Reply:
Thanks! Good suggestion, @buraizu .